### DISCUSSION TITLE
自动复制短信验证码

### DISCUSSION THREAD

**Discussion by GTX2077** [gkd-kit/gkd#694] [Ideas]
配合微信输入法的手机电脑同步剪切板

**Comment by Snoopy1866** [comment]
按以往经验，应该是获取不到短信通知浮窗的节点信息的

**Comment by AIsouler** [comment]
可以试试这个软件，几乎不耗电，耗电列表根本看不到它https://github.com/jd1378/otphelper

**Comment by adproqwq** [comment]
不会支持，你可以尝试上面推荐的软件
